frontend/src/components/P4LanguageServiceEditor.tsx

```diff
@@ -23,18 +28,28 @@ export default class MonacoEditor extends React.Component<MonacoEditorProps> {
     this.props.onChange(content)
   }
 
-  editorDidMount(editor: monaco.editor.IStandaloneCodeEditor):void{
-    this._editor = editorCreator(editor)
-    this.props.onMounted(this._editor) 
+//  editorWillMount(editor: monaco.editor.IStandaloneCodeEditor):void{
```

Comment:
Remove all the commented code, and the console.log entries for debugging.

Reply:
some were leftovers that were in the file before the PR, supposedly came into the file while testing out monaco standalone versions etc. However majority came from this change. Forgot to remove them. Took care of that now ;)

Reply:
I suggest to keep the console.log that will show that editor is about to be unmounted. This will call the disposal of the editor and consecutively the language client. Tracking down the problems with language server connects involved a lot of checking the console and using debugger and breakpoints in back- and frontend. I hope that it will not be necessary again, but I would keep some of the more status and less debugging oriented console outputs for now, as we should keep an eye on them while monaco languageclient is being reworked by typefox team and possibly new maintainer